src/app/notification/notification.service.ts

```diff
@@ -122,6 +122,12 @@ export class NotificationService {
             return;
         }
 
+        // Do not show programming error. They will be logged on the console window
+        if (error instanceof TypeError) {
+            this.warn("An unknown error has occurred while processing your request.");
```

Comment:
Instead of not showing anything from the original error message/detail, how about showing some part of it so that users can find what error happened.